hc/local_settings.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
You might need to change the name of this file
- It contains both local and Heroku settings 
- The name is deceptive